tools/mslite

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -e
+pushd $(cd $(dirname ${0})/..; pwd) > /dev/null
+
+bold() {
+    echo "$(tty -s && tput bold)$1$(tty -s && tput sgr0)"
+}
+
+clean() {
+    bold "mindspore clean"
+    rm -rf "./third_party/source/mindspore"
+}
+
+sync() {
+    bold "mindspore sync"
+    [ -d "./third_party/source/mindspore" ] || git clone --quiet https://github.com/mindspore-ai/mindspore.git "./third_party/source/mindspore"
+    pushd "./third_party/source/mindspore" > /dev/null
+    git pull --quiet --prune
+    popd > /dev/null
+}
+
+schema() {
+    echo "mslite schema"
+
+    # modify
+    cp ./third_party/source/mindspore/mindspore/lite/schema/*.fbs ./tools/
+    sed -i '1,16d' ./tools/model.fbs
```

Comment:
Why are schema file changes needed? Are these bugs in the schema file? Please fix and remove.